robotuille_domain.json

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
remove this from this PR, it does not belong here

Reply:
removed